### PR TITLE
First step in gutting ngi_reports

### PR DESCRIPTION
**Pull request by aanil**
* Removed code and references to ign reports
* Removed code and references to specifying node
* Removed reference to the statusdb module which we no longer use.

### CODE REVIEW

File: scripts/uppmax_ngi_reports.sh [hunk 1]
```diff
@@ -22,7 +22,7 @@ function make_report {
       fn=${f%.md}
       echo "Converting ${f}.."
       PD_DIR=${ngi_reports_dir}/data/pandoc_templates
-      ${PD_DIR}/pandoc --standalone --section-divs ${fn}.md -o ${fn}.html --template=${PD_DIR}/html_pandoc.html --default-image-extension=png --filter ${PD_DIR}/pandoc_filters.py -V template_dir=${PD_DIR}/ 
+      ${PD_DIR}/pandoc --standalone --section-divs ${fn}.md -o ${fn}.html --template=${PD_DIR}/html_pandoc.html --default-image-extension=png --filter ${PD_DIR}/pandoc_filters.py -V template_dir=${PD_DIR}/
```
Comment (ewels) [inline scripts/uppmax_ngi_reports.sh:25]: I guess this was auto formatting from your editor, but I think that these bash scripts can also be deleted. All they do is run pandoc I think.

File: scripts/ngi_reports [hunk 2]
```diff
@@ -106,57 +99,11 @@ def make_reports (report_type, ngi_node='unknown', working_dir=os.getcwd(), pand
...
 # calling main method to generate report
 if __name__ == "__main__":
     parser = argparse.ArgumentParser("Make an NGI Report")
     parser.add_argument('report_type', choices=allowed_report_types, metavar='<report type>',
         help="Type of report to generate. Choose from: {}".format(', '.join(allowed_report_types)))
-    parser.add_argument("-n", "--ngi_node", dest="ngi_node", default='unknown', choices=['unknown', 'stockholm', 'uppsala'],
-        help="NGI Node. Default: attempts to find out from config file or setup XML files.")
     parser.add_argument("-d", "--dir", dest="working_dir", default=os.getcwd(),
         help="Working Directory. Default: cwd when script is executed.")
     parser.add_argument("-l", "--pandoc_binary", dest="pandoc_binary", action='store_true',
```
Comment (ewels) [inline scripts/ngi_reports:109]: Can kill pandoc now 🔥

**Comment by aanil** [comment]
> Does it still work?

I ran it on a staging env in Irma and the output looked about the same.

File: scripts/uppmax_ngi_reports.sh (hunk 1 above)
Comment (aanil) [reply scripts/uppmax_ngi_reports.sh:25]: That's the next step! ✂️

**Comment by ewels** [comment]
Good enough for me! A bit tricky to review when you're just deleting code and moving it around tbh, but if it works then I'm happy.

**Pull request merged**